client/scss/components/_listing.scss

```diff
@@ -219,13 +225,17 @@ ul.listing {
         }
       }
     }
+
+    .icon.initial {
+      margin: 5px 0.3em 0 0;
+      vertical-align: top;
+    }
   }
 
   .actions {
-    @include clearfix();
-    margin-top: 0.8em;
-    margin-bottom: -0.5em;
+    float: right;
```

Comment:
Maybe this is just my lack of CSS-fu, but the only way I know of doing that is to move the `<ul class="actions">` before the `<div class="title-wrapper">` so that the title wraps around it - and I suspect that's an accessibility no-no (because we want screen-readers to read the title before the actions).

(Also, it's worth considering that wrapping onto a second line might happen because there are an excessive number of action buttons, and in that case it would make more sense to shift the buttons down. However, now that we've moved Edit into the More menu, and custom hooks / third-party packages are the only route by which we'll get any buttons here at all, this is probably enough of an edge case that we can disregard it...)